Makefile

```diff
@@ -0,0 +1,9 @@
+install_precommit:
```

Comment:
Could we make this `install_dev`, and have it install requirements_dev.txt? (and add precommit to that requirements file).
We are trying to remove all of the `pip install` commands we have scattered around

Reply:
good idea, will do that